exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -413,4 +413,8 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  String USE_DYNAMIC_UDFS = "exec.udf.use_dynamic";
```

Comment:
+ Looks like the above option "enables" dynamic UDFs, and this enables "using" dynamic UDFs. So why is this option required?
+ If this is intended to be a session option, and the above one is not, then make a check to ensure either of the options are enabled in QueryContext.